numpy/core/tests/test_ufunc.py

```diff
@@ -1915,3 +1915,24 @@ def test_invalid_args(self):
         exc = pytest.raises(TypeError, np.sqrt, None)
         # minimally check the exception text
         assert 'loop of ufunc does not support' in str(exc)
+
+    @pytest.mark.parametrize('nat', [np.datetime64('nat'), np.timedelta64('nat')])
+    def test_nat_is_not_finite(self, nat):
+        try:
+            assert not np.isfinite(nat)
+        except TypeError:
+            pass  # ok, just not implemented
+
+    @pytest.mark.parametrize('nat', [np.datetime64('nat'), np.timedelta64('nat')])
+    def test_nat_is_nan(self, nat):
+        try:
+            assert np.isnan(nat)
+        except TypeError:
+            pass  # ok, just not implemented
+
+    @pytest.mark.parametrize('nat', [np.datetime64('nat'), np.timedelta64('nat')])
+    def test_nat_is_no_tinf(self, nat):
```

Comment:
Made a typo here, oops. At any rate, let's let these run. Sadly, I'm expecting them to fail.

Reply:
Just changed `no_tinf -> not_inf`. Looks like the suite is passing.